src/index.html

```diff
@@ -17,11 +17,365 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="link-to-apple-page"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="link-to-samsung-page"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="link-to-smartphones-page"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="link-to-laptopscomputers-page"
+              class="nav__link"
+              data-qa="hover"
```

Comment:
The `data-qa` attribute for the 4th navigation link is incorrectly set to 'hover'. Please update it to the correct value as specified in the task requirements.